app/endpoints/campaign.py

```diff
@@ -320,6 +345,66 @@ async def update_list(
         raise HTTPException(status_code=400, detail=str(error))
 
 
+@router.get(
+    "/campaign/voters",
+    response_model=list[schemas_campaign.VoterGroup],
+    status_code=200,
+    tags=[Tags.campaign],
+)
+async def get_voters(
+    user: models_core.CoreUser = Depends(is_user_a_member),
+    db: AsyncSession = Depends(get_db),
+):
+    """
+    Return the voters (groups allowed to vorte) for the current campaign.
+    """
+    voters = await cruds_campaign.get_voters(db=db)
+    return voters
+
+
+@router.post(
+    "/campaign/voters",
+    response_model=list[schemas_campaign.VoterGroup],
+    status_code=201,
+    tags=[Tags.campaign],
+)
+async def add_voters(
+    voters: list[schemas_campaign.VoterGroup],
```

Comment:
I think it may be easier to pass a list of id directly.

For example:
```suggestion
    allowed_group_ids: list[str],
```

Reply:
We will pass a single VoterGroup